cross-region-replication-mm2-converter/src/main/java/com/amazonaws/services/crossregion/schemaregistry/kafkaconnect/CrossRegionReplicationMM2Converter.java

```
@@ -0,0 +1,162 @@
+        targetConfigs.put(AWSSchemaRegistryConstants.AWS_REGION, configs.get(AWSSchemaRegistryConstants.AWS_TARGET_REGION));
+
+        deserializationFacade =
+                GlueSchemaRegistryDeserializationFacade.builder()
+                        .credentialProvider(credentialsProvider)
+                        .configs(sourceConfigs)
+                        .build();
+
+        serializationFacade   =
+                GlueSchemaRegistrySerializationFacade.builder()
+                        .credentialProvider(credentialsProvider)
+                        .configs(targetConfigs)
+                        .build();
+
+        serializer   = new GlueSchemaRegistrySerializerImpl(credentialsProvider, new GlueSchemaRegistryConfiguration(targetConfigs));
+
+        deserializer = new GlueSchemaRegistryDeserializerImpl(credentialsProvider, new GlueSchemaRegistryConfiguration(sourceConfigs));
+
+    }
+
+    @Override
+    public byte[] fromConnectData(String topic, org.apache.kafka.connect.data.Schema schema, Object value) {
+        byte[] bytes = (byte[]) value;
+        if (value == null) return new byte[0];
+
+        try {
+            byte[] deserializedBytes = deserializer.getData(bytes);
+            Schema deserializedSchema = deserializer.getSchema(bytes);
+
+            byte[] encodedByte = serializer.encode(null, deserializedSchema, deserializedBytes);
+            com.amazonaws.services.schemaregistry.common.Schema returnedSchema = getSchema(bytes);
+            UUID uuid = registerSchema(returnedSchema);
+
+            return encodedByte;
+
+        } catch (SerializationException | AWSSchemaRegistryException e){
+            throw new DataException("Converting Kafka Connect data to byte[] failed due to serialization/deserialization error: ", e);
+        }
+
+    }
+
+    @Override
+    public SchemaAndValue toConnectData(String s, byte[] bytes) {
```

Let us add comment here stating this method should never be invoked since MM2 is a source connector. In addition to that, let us throw an exception instead of returning null